apps/website/src/components/content/Section.tsx

```diff
@@ -28,12 +28,14 @@ const Section = ({
     <section
       className={classes(
         offsetParent && "relative z-0",
-        dark ? "text-alveus-tan" : "text-alveus-green-900",
+        dark ? "text-alveus-tan" : "text-alveus-green-900 dark:text-white",
         // add vertical padding if not overwritten via className
         !/\bpy-\d+\b/.test(className || "") && "py-16",
         // add background color if not overwritten via className
         !/\bbg-/.test(className || "") &&
-          (dark ? "bg-alveus-green" : "bg-alveus-tan"),
+          (dark
+            ? "bg-alveus-green dark:bg-gray-800"
+            : "bg-alveus-tan dark:bg-gray-900"),
```

Comment:
I wonder if we can use greens, instead of greys (which are rather brown), to stay more aligned to alveus' normal color palette?